src/populateui.js

```diff
@@ -0,0 +1,53 @@
+const render = (taskList) => {
+  const sortedTask = taskList.list.sort((a, b) => a.index - b.index);
+  const taskContainer = document.querySelector('.lists');
+  let taskHtml = '';
+  sortedTask.forEach(({ completed, description, id }) => {
+    const checkedTodo = completed ? 'checked' : '';
+    const checkClass = completed ? 'checked' : '';
+    taskHtml += `  <div class="todo-item">
+                          <div>
+                              <input id="${id}" class="todo-check" type="checkbox" ${checkedTodo} />
+                              <input id="${id}" class="todo-edit ${checkClass}" type="text" value="${description}" />
+                          </div>
+                          <button id="${id}" class="remove-btn"><i id="remove" class="fa-solid fa-trash-can fa-lg"></i></button>
+                      </div>
+      `;
+  });
+
+  taskContainer.innerHTML = taskHtml;
+  const removeBtn = document.querySelectorAll('.remove-btn');
+  removeBtn.forEach((btn) => {
```

Comment:
- [ ] I don't see any `tasklist` array 🤔 Kindly as requested by the previous reviewer, make sure you follow the project  requirements:

- [ ] 1. You should have an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]. Your code might look like this:

   ```javascript
   // ⚠️ Note: Your code doesn't have to be implemented exactly like this, this is only "one" example of how it can be implemented.
   const tasks = [
     {
       description: 'An example description in string',
       completed: false,
       index: 1,
     },
   ];
   ```

- [ ] 2. You should have a function that iterates over the tasks array and populates the HTML list item element for each task. Your code might look like this:

   ```javascript   
   const populateTasks = (task, parentElement) => {
     const { description, completed, index: id } = task;
     const innerHTML = `
            <p>${description}</p>
                  .......`;
     parentElement.appendChild(newTodoListElement);
   };
   ```
- Remember. On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task. You have to make sure the first index is 1 and not zero this way when you create a todo list, the index starts from 1 instead of 0